src/test/ReactTestUtils.js

```diff
@@ -500,6 +500,7 @@ function makeSimulator(eventType) {
       fakeNativeEvent,
       node
     );
+    event.persist();
```

Comment:
We shouldn't do this. It would make `Simulate.*` event handlers behave differently than they would in the browser where events aren't automatically persisted (eg a handler calls something with the event in a timeout would end up working in tests but fail in real life).

We should fix that and any other tests, something like this.

```js
var foo = {
  onChange = function(e) {
    e.persist();
  }
}
spyOn(foo, 'onChange').andCallThrough();
```